eos-paygd/main.c

```diff
@@ -271,6 +272,14 @@ test_and_update_securitylevel (void)
   return TRUE;
 }
 
+static gint
+sort_func (gconstpointer p1, gconstpointer p2)
+{
+  const gchar *a = (const char *) p1;
+  const gchar *b = (const char *) p2;
+  return g_strcmp0 (a, b);
+}
```

Comment:
You don't need this, you can just pass `g_strcmp0` as the compare_func.

Reply:
Cast it: `(GCompareFunc) g_strcmp0`

Reply:
:brain: